ext/pdo/pdo_stmt.c

```diff
@@ -693,7 +693,7 @@ static bool do_fetch(pdo_stmt_t *stmt, zval *return_value, enum pdo_fetch_type h
 	if (how == PDO_FETCH_COLUMN) {
 		int colno = stmt->fetch.column;
 
-		if ((flags & PDO_FETCH_GROUP) && stmt->fetch.column == -1) {
+		if ((flags & (PDO_FETCH_GROUP|PDO_FETCH_UNIQUE)) && stmt->fetch.column == -1) {
```

Comment:
Can you please explain this change?

Reply:
Currently, `PDO_FETCH_UNIQUE` overlaps with `PDO_FETCH_GROUP`. Now, "properly" split as bit flags means these paths are no longer taken if `PDO::FETCH_UNIQUE` is used.